app/controllers/api/v1/admin/problems_controller.rb

```diff
@@ -34,7 +34,7 @@ def index
         end
 
         def problems_list
-          problems = Problem.all.order('title')
+          problems = Problem.kept.all.order('title')
```

Comment:
```suggestion
          problems = Problem.kept.order('title')
```
I think .all won't be necessary here